src/models/modelUtils.js

```diff
@@ -26,7 +26,7 @@ export const validImageURL = (val) => {
 };
 
 export const validApplyURL = (val) => {
-    const httpRegex = /^https?:\/\/(www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b([-a-zA-Z0-9()@:%_+.~#?&//=]*)?$/;
+    const httpRegex = /(^https?:\/\/)\S+\.\S+$/;
```

Comment:
```suggestion
    const httpRegex = /^https?:\/\/\S+\.\S+$/;
```
I don't see how a group is needed here but I might be missing something, if that's the case just mark this as resolved.

Reply:
I just grouped to organize it

Reply:
@CarlosMealha Just remove those brackets and then we can merge this